dev-manuals/test-plans/overworld-frontend/plans/minimap-functions.md

```diff
@@ -0,0 +1,41 @@
+# Name: minimap functions
+
+Number: U.hud.1
+Version: V0.1, 2022-08-10, create test case
+Author: Michael Linder
+Tester: -
+
+## Description
+
+This use case verifies that the minimap works as intended.  
+
+## Precondition
+
+Overworld is loaded, minimap is displayed
+
+## Postcondition
+
+## Typical procedure
+
+1. The tester checks if the Player head is always in the middle of the minimap by walking around
+2. The tester zooms in the minimap with the keybind
+3. The tester zooms out the minimap with the keybind
+4. The tester zooms in the minimap with the button
+5. The tester zooms out the minimap with the button
+6. The tester checks if there are always 4 zoom levels
+7. The tester walks around and checks if the minimap displays npc heads and area symbols
+8. The tester zooms out the game with the keybind and checks if the minimap also zoomed out
+9. The tester zooms in the game with the keybind and checks if the minimap also zoomed in
```

Comment:
Isn't that already in another test plan?

Reply:
yes, but rather be safe then sorry no?

Reply:
Why test it twice?
Doesn't that only waste time?

Reply:
Especially if we test for multiple OSses?